llama-stack/helm/values.yaml

```diff
@@ -44,8 +44,10 @@ env:
     value: "4096"
   - name: VLLM_API_TOKEN
     value: fake
-  - name: OTEL_ENDPOINT
-    value: http://otel-collector-collector.observability-hub.svc.cluster.local:4318/v1/traces
+  - name: OTEL_EXPORTER_OTLP_ENDPOINT
+    value: http://otel-collector-collector.observability-hub.svc.cluster.local:4318
```

Comment:
@fax4ever what happens if this ep doesn't exist? should we control this with a flag in values.yaml?

Reply:
good idea!